asr1k_neutron_l3/models/neutron/l3/interface.py

```diff
@@ -90,6 +90,7 @@ def _primary_subnet(self):
         for subnet in self.router_port.get('subnets', []):
             if subnet.get('id') == self._primary_subnet_id:
                 return subnet
+        return None
```

Comment:
Wouldn't this return `None` either way?

Reply:
I once read in a style guide that its good style that if your method returns anything that it should have a return case for every case, also for the implicit `return None`. The idea behind this is to either have methods that return nothing (`None`) and then have methods that remind you that they return something in every case.

In this case it felt a bit like we ignored the default case, so I added an explicit return. Not important to keep it, just a thought :)